### ISSUE TITLE
Fix failing Windows tests

### ISSUE THREAD

**Issue by hudochenkov** [stylelint/stylelint#4337]
Following discussion started in https://github.com/stylelint/stylelint/issues/4333. Failing tests are caused by Github Actions inconsistent environment. I've contacted support and they told me to open an issue, which I did https://github.com/actions/virtual-environments/issues/33.

This issue is to have reference about status of tests.

**Comment by ntwb** [comment]
Related: Windows CI tests are switching to PowerShell by default next week

https://github.blog/changelog/2019-10-17-github-actions-default-shell-on-windows-runners-is-changing-to-powershell/

> If your `run` step contains Windows batch scripting you should update it to specify `cmd` as the shell to avoid being broken.

```shell
- run: |
    ...
  shell: cmd
```

As we are not running any batch scripting, in my mind it should not break!

**Comment by XhmikosR** [comment]
Do we know if this issue still happens?

**Comment by hudochenkov** [comment]
It's not happening anymore, but the fix is ugly :(

**Comment by XhmikosR** [comment]
No, I meant if the issue happens without the workaround :)

**Comment by hudochenkov** [comment]
We need to remove workaround and check.

**Comment by XhmikosR** [comment]
I have a branch, I'll try to clean it up and see how it goes.
